kibana/examples/openshift/Makefile

```diff
@@ -0,0 +1,15 @@
+default: test
+include ../../../helpers/examples.mk
+
+RELEASE := kibana
+
+template:
+	helm template --values ./values.yml ../../
+
+install: 
+	helm upgrade --wait --timeout=600 --install --values ./values.yml $(RELEASE) ../../ ; \
```

Comment:
```suggestion
	helm upgrade --wait --timeout=600 --install --values ./values.yml $(RELEASE) ../../
```

Reply:
Fixed! Weird how committing your suggestion still requires you to re-review it haha